automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupApiSpec.scala

```diff
@@ -150,6 +154,31 @@ class AuthDomainGroupApiSpec extends AnyFreeSpec with Matchers with WorkspaceFix
       }
 
     }
+
+    "bucket should not be accessible to project owners via projectViewer Google role" in {
+
+      val userA = UserPool.chooseProjectOwner //The project owner who can't see the workspace
+      val userB = UserPool.chooseAuthDomainUser //The user who owns the workspace
+
+      val userAToken: AuthToken = userA.makeAuthToken()
+      val userBToken: AuthToken = userB.makeAuthToken()
+
+      withGroup("AuthDomain", List(userB.email)) { authDomainName =>
+        withCleanBillingProject(userA, List(userA.email), List(userB.email)) { projectName =>
+          withWorkspace(projectName, "AuthDomainGroupApiSpec_workspace", Set(authDomainName)) { workspaceName =>
+
+            val bucketName = Rawls.workspaces.getWorkspaceDetails(projectName, workspaceName)(userBToken).parseJson.convertTo[WorkspaceResponse].workspace.bucketName
+
+            eventually {
+              //assert that userA receives 403 when trying to access bucket
+              Google.storage.getBucket(bucketName)(userAToken).status.intValue() should be(403)
```

Comment:
For good measure - do we also need to make sure that `userB` _can_ access the bucket?